README.md

```diff
@@ -1540,23 +1530,24 @@ JSON encoded body using the "application/json" content type.
     <td>optional</td>
   </tr>
   <tr>
-    <td>snapshot</td>
-    <td>element</td>
-    <td>Picture of the viewpoint</td>
+    <td>bitmaps</td>
```

Comment:
Should bitmaps instead be singular since Bitmap GET and PUT only supports one bitmap pr. viewpoint?